<ui version="4.0" >
 <class>DefaultToolWidget</class>
 <widget class="QTabWidget" name="DefaultToolWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>354</height>
   </rect>
  </property>
  <property name="currentIndex" >
   <number>0</number>
  </property>
  <widget class="QWidget" name="geometryTab" >
   <attribute name="title" >
    <string>Geometry</string>
   </attribute>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QGroupBox" name="positionGroup" >
      <property name="title" >
       <string>Position</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignHCenter</set>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" colspan="2" >
        <widget class="KoPositionSelector" native="1" name="positionSelector" >
         <property name="minimumSize" >
          <size>
           <width>75</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QLabel" name="positionXLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Position x:</string>
         </property>
         <property name="buddy" >
          <cstring>positionXSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2" >
        <widget class="KoUnitDoubleSpinBox" name="positionXSpinBox" >
         <property name="minimum" >
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QLabel" name="positionYLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Position y:</string>
         </property>
         <property name="buddy" >
          <cstring>positionYSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2" >
        <widget class="KoUnitDoubleSpinBox" name="positionYSpinBox" >
         <property name="minimum" >
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QGroupBox" name="dimensionGroup" >
      <property name="title" >
       <string>Dimensions</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignHCenter</set>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="widthLabel" >
         <property name="text" >
          <string>Width:</string>
         </property>
         <property name="buddy" >
          <cstring>widthSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item rowspan="2" row="0" column="1" >
        <widget class="KoAspectButton" native="1" name="aspectButton" >
         <property name="minimumSize" >
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>10</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="KoUnitDoubleSpinBox" name="widthSpinBox" >
         <property name="minimum" >
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="heightLabel" >
         <property name="text" >
          <string>Height:</string>
         </property>
         <property name="buddy" >
          <cstring>heightSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="KoUnitDoubleSpinBox" name="heightSpinBox" >
         <property name="minimum" >
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="advancedTab" >
   <attribute name="title" >
    <string>Advanced</string>
   </attribute>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="orderLabel" >
      <property name="text" >
       <string>Order:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QToolButton" name="bringToFront" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2" >
     <widget class="QToolButton" name="raiseLevel" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3" >
     <widget class="QToolButton" name="lowerLevel" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="4" >
     <widget class="QToolButton" name="sendBack" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5" >
     <widget class="QGroupBox" name="alignmentGroup" >
      <property name="title" >
       <string>Alignment</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignHCenter</set>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="horizontalLabel" >
         <property name="text" >
          <string>Horizontal:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QToolButton" name="leftAlign" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QToolButton" name="hCenterAlign" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QToolButton" name="rightAlign" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="verticalLabel" >
         <property name="text" >
          <string>Vertical:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QToolButton" name="topAlign" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QToolButton" name="vCenterAlign" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="3" >
        <widget class="QToolButton" name="bottomAlign" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="5" >
     <widget class="QGroupBox" name="transformationGroup" >
      <property name="title" >
       <string>Transformation</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignHCenter</set>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QComboBox" name="rotateCombo" />
       </item>
       <item row="0" column="1" >
        <widget class="QPushButton" name="rotateButton" >
         <property name="text" >
          <string>Rotate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QComboBox" name="scaleCombo" />
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="scaleButton" >
         <property name="text" >
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QComboBox" name="shearCombo" />
       </item>
       <item row="2" column="1" >
        <widget class="QPushButton" name="shearButton" >
         <property name="text" >
          <string>Shear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoPositionSelector</class>
   <extends>QWidget</extends>
   <header>KoPositionSelector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KoAspectButton</class>
   <extends>QWidget</extends>
   <header>KoAspectButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KoUnitDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>KoUnitDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
